scan.php

```diff
@@ -175,38 +175,38 @@ function patch_file($file, $content)
         foreach ($files as $file) {
             if (is_dir($file) === true) { // Not in use, was used to check directory traversal was working properly
             } else { // If is file
-                        if (preg_match("/\.php$/", $file)) { // Currently only selects PHP scripts for scanning
```

Comment:
As usual: Undo your commit. Add your tab modification with `git add --patch`, commit, then commit again for the rest. ;-)